v21.2/transactions.md

```diff
@@ -125,6 +125,10 @@ transaction to retry, the client needs to be involved in retrying the whole
 transaction and so you should write your transactions to use
 [client-side intervention](#client-side-intervention).
 
+#### Bounded staleness reads
+
+In the event [bounded staleness reads](follower-reads.html#bounded-staleness-reads) are used along with either the `with_min_timestamp` or `with_max_staleness` functions and the `nearest_only` parameter set to `true`, the query will throw an error if it can't be served by a nearby replica. 
```

Comment:
Only suggestion is linking the functions to this page: https://www.cockroachlabs.com/docs/v21.2/functions-and-operators#date-and-time-functions. LGTM otherwise!

Reply:
Added. Thank you!